app/controllers/mailgun_webhooks_controller.rb

```diff
@@ -0,0 +1,45 @@
+require "openssl"
+
+class MailgunWebhooksController < ApplicationController
+  skip_before_action :redirect_to_getyourrefund
+  skip_before_action :set_visitor_id
+  skip_before_action :set_source
+  skip_before_action :set_referrer
+  skip_before_action :set_utm_state
+  skip_before_action :set_sentry_context
+  skip_before_action :check_maintenance_mode
+  skip_after_action :track_page_view
+  skip_before_action :verify_authenticity_token
```

Comment:
would love to refactor this inheritance and all these before_actions. Also present in TwilioWebhooksController, and would be useful in ZendeskWebhooks

Reply:
Should we?
1. do a base WebhookController that skips all the junk
2. have webhook controllers inherit from `ActionController::Base`
3. make a new base controller for all non-webhook controllers that does all these before actions

Reply:
Jenny says 1 or 2, preferably 2

I say 4 or 2, preferably 4, but 2 is also great

sounds like 2 :P